client/src/api/TourEditService.ts

```diff
@@ -54,7 +58,7 @@ export function uploadMapImage(sessionId, file, width, height) {
     });
 }
 
-export function updateImage360(sessionId, placeId, file, width, height) {
+export function updateImage360(sessionId: string, placeId: string, file, width: string, height: string) {
```

Comment:
I think width and height are numbers
![image](https://user-images.githubusercontent.com/9947582/57574258-5b4bcf00-743e-11e9-824e-65bf6e1b9385.png)